.changeset/tame-buckets-end.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-api-docs': minor
```

Comment:
Considering the contributions guide, I think it is a patch:
https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#how-to-create-a-changeset